src/mudlet.cpp

```diff
@@ -1538,7 +1542,7 @@ void mudlet::addConsoleForNewHost(Host* pH)
     setWindowTitle(pH->getName() + " - " + version);
 
     mpSplitter_profileContainer->addWidget(pConsole);
-    if (mpCurrentActiveHost) {
+    if (mpCurrentActiveHost && !mMultiView) {
```

Comment:
This was what was causing the currently active profile to not be shown after a new one was opened when multi-view mode was enabled...!